src/vivarium_census_prl_synth_pop/components/person.py

```diff
@@ -175,12 +184,18 @@ def on_time_step(self, event: Event) -> None:
     ##################
 
     def _get_new_household_ids(
```

Comment:
You can pass the whole `persons` dataframe as a parameter (avoiding the need to call `self.population_view.get()` on line 191) and the index of people who are moving. 

This entire method is arguably unnecessary to split out from the calling location, as it's not being reused.